tools/editor/project_manager.cpp

```diff
@@ -692,6 +692,10 @@ void ProjectManager::_unhandled_input(const InputEvent& p_ev) {
 				}
 
 			} break;
+            case KEY_F: {
```

Comment:
Could you change your indentation parameters to use tabs like in the rest of the source? And then ideally amend this commit (`git commit --amend`) to fix it.